__tests__/index_test.re

```diff
@@ -0,0 +1,17 @@
+/* Reason */
+open Jest;
+
+let () =
+  describe("Expect", () =>
+    Expect.(test("toBe", () =>
+              expect(1 + 2) |> toBe(3)
```

Comment:
an intended space? :thinking:

Reply:
yo

Reply:
@muhammadridho  is it okay?

Reply:
i think we need prettier, it is because reformatter from reasonml